protostar-rust/tests/e2e/running.rs

```diff
@@ -14,22 +14,68 @@ fn run_simple_test() {
         .current_dir(&temp)
         .assert()
         .success()
-        .stdout_matches(indoc! {r#"Collected 6 test(s) and 3 test file(s)
+        .stdout_matches(indoc! {r#"Collected 8 test(s) and 4 test file(s)
             Running 1 test(s) from src/lib.cairo
-            [PASS] [..]::test_fib
+            [PASS] src::test_fib
             Running 2 test(s) from tests/ext_function_test.cairo
             [PASS] ext_function_test::ext_function_test::test_my_test
             [PASS] ext_function_test::ext_function_test::test_simple
-            Running 3 test(s) from tests/test_simple.cairo
+            Running 4 test(s) from tests/test_simple.cairo
             [PASS] test_simple::test_simple::test_simple
             [PASS] test_simple::test_simple::test_simple2
+            [PASS] test_simple::test_simple::test_two
             [FAIL] test_simple::test_simple::test_failing failing check
-            Tests: 5 passed, 1 failed
+            Running 1 test(s) from tests/without_prefix.cairo
+            [PASS] without_prefix::without_prefix::five
+            Tests: 7 passed, 1 failed
         "#});
 }
 
 #[test]
-fn run_declare_test() {
+fn with_filter() {
```

Comment:
Do we have a test for exact match?

Reply:
Only unit, I'll add an e2e test. 

https://github.com/software-mansion/protostar/blob/1999-selecting-tests/protostar-rust/src/lib.rs#L506